react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
So what do you think of the following approach:

on `componentDidMount` ([life cycle hook](https://facebook.github.io/react/docs/react-component.html#componentdidmount)) the `SearchBarComponent` parses the url, gets the query and sets its initial value to the value of the query? Then we can remove this component together with dataForReact and I feel like it's cleaner and centralized?